tests/hipscat_import/test_map_reduce.py

```diff
@@ -201,7 +201,7 @@ def test_reduce_order0(parquet_shards_dir, assert_parquet_file_ids, tmp_path):
         id_column="id",
     )
 
-    output_file = os.path.join(tmp_path, "Norder0/Npix11", "catalog.parquet")
+    output_file = os.path.join(tmp_path, "Norder=0/Dir=0/Npix=11.parquet")
```

Comment:
I think when doing `os.path.join` each directory should be an argument to the method rather than having the slashes in the string. This is just for windows support though which we probably don't do anywhere else anyway.